README.md

```diff
@@ -60,18 +57,12 @@ It can be used as a component of a frontend such as
 interface for wallets. Most users who would like to use Cardano should
 start with Daedalus.
 
-> :information_source: This source code repository contains the next major version of Cardano
-> Wallet, which has been completely rewritten for the
-> [Shelley](https://roadmap.cardano.org/) phase.
->
-> :bulb: The Byron version of Cardano Wallet is in the
-> [cardano-sl](https://github.com/input-output-hk/cardano-sl)
-> repository.
+## Quickstart
 
-## Getting Started
+The `cardano-wallet` executable is a HTTP server which manages your wallet(s). Here is one way to start the server using Docker:
```

Comment:
Hi @HeinrichApfelmus! 

I would suggest using "that" here instead of "which", since the ability to manage wallets is a defining quality of the wallet software, and not something it just happens to do (in addition to being an HTTP server). (I think you alluded to this in a separate comment!)

Additionally, would recommend changing "a HTTP server" to "an HTTP server":

```suggestion
The `cardano-wallet` executable is an HTTP server that manages your wallet(s). Here is one way to start the server using Docker:
```

Reply:
Ah, it looks like I mixed up `that` and `which`. Will do.